src/index.ts

```diff
@@ -1,17 +1,27 @@
 import Vue from 'vue'
 import VueI18n from 'vue-i18n'
-import { computed, getCurrentInstance } from '@vue/composition-api'
+import { computed, getCurrentInstance, WritableComputedRef } from '@vue/composition-api'
+import { VueConstructor } from 'vue/types/umd'
 
 let i18nInstance: VueI18n | undefined
 
-export function createI18n(options?: VueI18n.I18nOptions) {
-  Vue.use(VueI18n)
+export function createI18n(options?: VueI18n.I18nOptions, vue: VueConstructor = Vue): VueI18n {
+  vue.use(VueI18n)
   i18nInstance = new VueI18n(options)
 
   return i18nInstance
 }
 
-export function useI18n() {
+export interface I18nHook {
```

Comment:
The interface name should also be `Composer` , seamless migration to vue-i18n-next.

ref: https://github.com/intlify/vue-i18n-next/blob/5a81eeb42caf8a272af47f8a01d2021635f6d2ca/src/composer.ts#L325-L909

Reply:
renamed

Reply:
👍